types/sticky-cluster/sticky-cluster-tests.ts

```diff
@@ -0,0 +1,34 @@
+import * as stickyCluster from 'sticky-cluster';
```

Comment:
If I'm right, since `stickyCluster` is function, it should be `import stickyCluster = require('sticky-cluster');` here.

Reply:
@plantain-00 - I was unaware about that convention for function imports - I will go do some research to get up to speed.

Thank you for taking the time to review, I have updated the test case with the correct import statement.

Reply:
this link should help: https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this